### ISSUE TITLE
[Feature] Limit Break

### ISSUE THREAD

**Issue by mkaito** [cairthenn/Redirect#14]
Allow setting LB target priorities to skip manual ground targeting, as well as queueing.

**Comment by cairthenn** [comment]
I agree this would be useful, but can't think of a particularly convenient way to test it, so it'll probably take me some time to see if it can be done

**Comment by PrincessRTFM** [comment]
Entering dungeons in explorer mode reportedly allows you to freely LB at all three ranks

**Comment by Nox13last** [comment]
> Entering dungeons in explorer mode reportedly allows you to freely LB at all three ranks

Having just wiffed a BLM LB against MSQ Ultima Weapon stage 1 by targeting myself, I did just this. 
You get a little window (similar to the GC Squadron Orders window) with four buttons. One for each LB tier, and one for summoning a targetable training dummy (which works identically to the dummies you find in wilderness settlements or on estate lawns).
With "Place all ground targets at the cursor" set to true, any attempt to use a ground-target LB (BLM's _Skyshard_) just returns an "Invalid Target" action error with no context. If your LB is part of a macro (Explorer Mode cannot test for this), and it uses a placeholder (such as _<t>_), then it will target you without erroring-out. And make you look dumb.
